### PR TITLE
Feature/creative commons

### PR DESCRIPTION
**Pull request by FranFin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,23 @@ See [editing instructions](EDITING-INSTRUCTIONS.md)
 
 ## License
 
-The content of this repository and the generated website is open-sourced under the Apache-2.0 license. See the [LICENSE](LICENSE) file for details. Please note the following third-party elements and content:
+The content of this repository and the generated website is open-sourced under the Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0) license.
+You are free to:
+
+**Share** — copy and redistribute the material in any medium or format for any purpose, even commercially.
```
Comment (ralph-lange) [inline README.md:17]:
```suggestion
* **Share** — copy and redistribute the material in any medium or format for any purpose, even commercially.
```

File: README.md [hunk 2]
```diff
@@ -11,7 +11,23 @@ See [editing instructions](EDITING-INSTRUCTIONS.md)
...
+The licensor cannot revoke these freedoms as long as you follow the license terms.
+
+Under the following terms:
+
+**Attribution** — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
```
Comment (ralph-lange) [inline README.md:22]:
```suggestion
* **Attribution** — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
```

File: README.md [hunk 3]
```diff
@@ -11,7 +11,23 @@ See [editing instructions](EDITING-INSTRUCTIONS.md)
...
+
+**NoDerivatives** — If you remix, transform, or build upon the material, you may not distribute the modified material.
```
Comment (ralph-lange) [inline README.md:24]:
```suggestion
* **NoDerivatives** — If you remix, transform, or build upon the material, you may not distribute the modified material.
```

File: README.md [hunk 4]
```diff
@@ -11,7 +11,23 @@ See [editing instructions](EDITING-INSTRUCTIONS.md)
...
+
+**No additional restrictions** — You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits.
```
Comment (ralph-lange) [inline README.md:26]:
```suggestion
* **No additional restrictions** — You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits.
```
..., i.e. make the restrictions a list.

**Review by ralph-lange — CHANGES_REQUESTED** [review]

**Review by ralph-lange — APPROVED** [review]

**Review by pablogs9 — APPROVED** [review]

**Pull request merged**